ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -5,10 +5,10 @@
 
 import UIKit
 
-class TabBarItemView: UIView {
+public class TabBarItemView: UIView {
```

Comment:
Is this turning public just because of the Demo app?
If that's the only goal, I think we should find a way around it so it does not add up on the final binary size.

Reply:
changing direction and remove this file change